### PR TITLE
Update to dragging intermediate state when state changes are animated

### PR DESCRIPTION
**Pull request by shagedorn**

### CODE REVIEW

File: ISHPullUp/ISHPullUpViewController.m [hunk 1]
```diff
@@ -390,7 +401,10 @@ - (void)setBottomHeight:(CGFloat)bottomHeight animated:(BOOL)animated {
           initialSpringVelocity:config.initialVelocity
                         options:config.options
                      animations:updateBlock
-                     completion:nil];
+                     completion:^(BOOL finished) {
+                         self.isAnimatingStateChange = NO;
```
Comment (felixLam) [inline ISHPullUp/ISHPullUpViewController.m]: only if finished?

Comment (shagedorn) [inline ISHPullUp/ISHPullUpViewController.m]: Not really. If it hasn't finished, it still means it is not moving anymore.

Comment (felixLam) [inline ISHPullUp/ISHPullUpViewController.m]: Ok, it's probably better to falsely set it back a bit too early than to never set it back.

**Comment by felixLam** [comment]
:shipit:

**Pull request merged**